src/App.tsx

```diff
@@ -417,12 +419,18 @@ function App() {
 
     useEffect(() => {
         const { section } = content[currentModule][page];
-        if (section === Section.Experiment) {
+        if (
+            section === Section.Experiment &&
+            timeFactor !== LiveSimulationData.DEFAULT_TIME_FACTOR
+        ) {
             setTimeFactor(LiveSimulationData.DEFAULT_TIME_FACTOR);
-        } else if (section === Section.Introduction) {
+        } else if (
+            section === Section.Introduction &&
+            timeFactor !== LiveSimulationData.INITIAL_TIME_FACTOR
+        ) {
```

Comment:
these changes aren't directly connected, but I did realize i was doing more updates to state than I needed to, which became more of an issue when i'm pushing the rendering